kaldb/src/test/java/com/slack/kaldb/logstore/search/LogIndexSearcherImplTest.java

```diff
@@ -382,26 +382,112 @@ public void testFullTextSearch() {
         makeMessageWithIndexAndTimestamp(1, "apple", TEST_DATASET_NAME, time.plusSeconds(4));
     msg1.addProperty("field1", "1234");
     strictLogStore.logStore.addMessage(msg1);
+    strictLogStore.logStore.commit();
+    strictLogStore.logStore.refresh();
+    // Search using _all field.
+    assertThat(
```

Comment:
can we add a test with multiple words with spaces and some numbers etc ?

Reply:
Let's also go ahead and add tests for empty string and wildcard as well.

Reply:
Done.